src/main/java/org/datadog/jenkins/plugins/datadog/clients/DatadogApiClient.java

```diff
@@ -364,9 +364,8 @@ public TraceWriteStrategy createTraceWriteStrategy() {
 
         Map<String, String> headers = Map.of(
             "DD-API-KEY", Secret.toString(apiKey),
-            "DD-CI-PROVIDER-NAME", "jenkins",
-            "Content-Encoding", "gzip");
-        JsonPayloadSender<Payload> payloadSender = new CompressedBatchSender<>(httpClient, url, headers, PAYLOAD_SIZE_LIMIT, p -> p.getJson());
+            "DD-CI-PROVIDER-NAME", "jenkins");
+        JsonPayloadSender<Payload> payloadSender = new BatchSender<>(httpClient, url, headers, PAYLOAD_SIZE_LIMIT, p -> p.getJson(), true);
```

Comment:
```suggestion
        JsonPayloadSender<Payload> payloadSender = new BatchSender<>(httpClient, url, headers, PAYLOAD_SIZE_LIMIT, p -> p.getJson(), compressionEnabled);
```

Similar comment here.

Reply:
Created a named constant